apps/platform-app/src/app/[locale]/(profile)/_actions/queries/createUsername.ts

```diff
@@ -0,0 +1,34 @@
+import { findLinkedinUsernamesByUsernameBase } from '@/backend/linkedin-details/linkedin-details.service'
```

Comment:
**nit:** It's not an action - let's put it to `user.service.ts`. Actions in our system are business oriented operations that always return domain specific models. But remember to avoid cross-imports. I mean, `.service` logic should not import `_actions` logic. Only one way, so actions import functions from services